ra-tls/Makefile

```diff
@@ -19,19 +19,15 @@ ifdef ECDSA
   SGX_DCAP_URI := https://github.com/intel/SGXDataCenterAttestationPrimitives
   SGX_DCAP_COMMIT := bfab1376480f760757738092399d0d99b22f4dfd
   SGX_DCAP ?= SGXDataCenterAttestationPrimitives
-  SGX_DCAP_INC := -I$(SGX_DCAP)/QuoteGeneration/quote_wrapper/common/inc -I$(SGX_DCAP)/QuoteGeneration/pce_wrapper/inc -I$(SGX_DCAP)/QuoteVerification/Src/AttestationLibrary/include
+  SGX_DCAP_INC := -I$(SGX_DCAP)/QuoteGeneration/quote_wrapper/common/inc -I$(SGX_DCAP)/QuoteGeneration/pce_wrapper/inc -I$(SGX_DCAP)/QuoteVerification/Src/AttestationLibrary/include -I$(SGX_DCAP)/QuoteVerification/QVL/Src/AttestationLibrary/include/
```

Comment:
The trailing "/" is not necessary.

Reply:
Plz split this single line into multi-lines.

Reply:
done